### PR TITLE
tlt-2670: cross_list_courses - selenium test updates

### PR DESCRIPTION
**Pull request by elliottyates**
fixes existing selenium tests, adds search capability test

@hktest @sapnamysore

### CODE REVIEW

File: selenium_tests/cross_listing/page_objects/cross_listing_main_page.py [hunk 1]
```diff
@@ -35,11 +38,19 @@ def ERROR_TEXT_LOCATOR(cls, expected_text):
 class MainPageObject(CrossListingBasePageObject):
     page_loaded_locator = Locators.HEADING_ELEMENT
 
+    # todo: consider moving into the BasePageObject, with override flexibility
+    def __init__(self, driver):
```
Comment (hktest) [inline selenium_tests/cross_listing/page_objects/cross_listing_main_page.py]: Understood how you are using it, but not understanding the __init__reference here. Will ping you later. :)

**Comment by hktest** [comment]
just a question I'll take offline. but :+1,  thanks!

File: selenium_tests/cross_listing/remove_cross_listing_tests.py [hunk 2]
```diff
@@ -24,6 +26,7 @@ def test_remove_mapping(self, test_case_id, primary_cid,
         avoid incidental conflict when we try to add a new mapping.
         2.  Add cross-listing via rest api of the pairing that we
         want to test the 'remove' functionality on.
+        3.  Find this new cross-listing using the search box
         3.  Test remove by clicking on the cross-list remove icon via UI
```
Comment (sapnamysore) [inline selenium_tests/cross_listing/remove_cross_listing_tests.py]: nitick: increment 3 to 4?

File: selenium_tests/cross_listing/add_cross_listing_tests.py [hunk 3]
```diff
@@ -11,44 +11,46 @@ class AddCrossListingTests(CrossListingBaseTestCase):
 
     @data(*get_xl_data(TEST_DATA_CROSS_LISTING_MAPPINGS))
     @unpack
-    def test_add_cross_listing_pairing(self, test_case,
+    def test_add_cross_listing_pairing(self, unused_test_case_id,
```
Comment (sapnamysore) [inline selenium_tests/cross_listing/add_cross_listing_tests.py]: Curious about the name 'unused_test_case_id'?

Comment (elliottyates) [inline selenium_tests/cross_listing/add_cross_listing_tests.py]: Yes -- unfortunately (?) python doesn't allow the use of multiple throwaway `_` variables in the same function signature, so one recommendation I ran across was to prefix with `unused_`. But apparently [PyLint doesn't like that](http://stackoverflow.com/questions/10107350/how-to-handle-the-pylint-message-idw0612-unused-variable), so maybe this should ultimately be changed to something like `_unused_`. Probably not an important update for this PR, but a thought for the future.

File: selenium_tests/cross_listing/remove_cross_listing_tests.py (hunk 2 above)
Comment (elliottyates) [inline selenium_tests/cross_listing/remove_cross_listing_tests.py]: yep

**Pull request merged**